guides/doc-Provisioning_Guide/topics/Virt-VMWare.adoc

```diff
@@ -288,7 +288,12 @@ You can use the same procedures to set up a VMware compute resource within {Proj
 
 .VMware cloud-init Provisioning Overview
 
+ifeval::["{build}" == "satellite"]
 image:ProvisioningUserData-Sequence.png[]
+endif::[]
+ifeval::["{build}" != "satellite"]
+image:provisioning_user_data_sequence.svg[]
+endif::[]
```

Comment:
One thing - we are setting a standard here - we should probably figure out some naming convention to distinquish upstream and satellite. How about `user-data-sequence-foreman.svg` and `user-data-sequence-satellite.{svg,png}`?

Then we should probably update the README with some instructions about how to properly name diagramy. Would you mind doing that?

Reply:
This would be the behaviour of a group following a standard! +1 from me.

Reply:
I like the idea of simply suffixing `-foreman` and `-satellite`. I prefer underscores but can live with dashes, too. 

I've reworked this PR and added a new convention.

Reply:
Dashes was just an idea, I don't mind I just saw the "Virt-VMWare.adoc" filename here and I thought that's the preferred way in this repo.